src/main/java/uk/go/hm/icb/service/AbstractSearchService.java

```diff
@@ -24,7 +24,7 @@ protected ICBResponse buildBaseResponse(ICBResponse.ICBResponseBuilder responseB
         if (matches.isEmpty()) {
             return buildNoMatchResponse(responseBuilder);
         } else if (matches.size() == 1) {
-            return matchResponseBuilder.buildSingleMatchResponse(responseBuilder, matches.getFirst(), request);
+            return matchResponseBuilder.buildSingleMatchResponse(responseBuilder, matches.get(0), request);
```

Comment:
I think getFirst() enhances readability and intent

Reply:
I fully agree, but, I use java17 (for crpf) which doesn't support getFirst(), can we use get(0) if that's ok? trying to avoid hazzle of me maintaining multiple java versions :)